src/App.jsx

```diff
@@ -1,28 +1,93 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
+import getResponse from './api/api';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+const url = 'https://mate-api.herokuapp.com/todos';
```

Comment:
URL of API server in two files at the same time, here and in User component. If address of server will change it can lead to error